modules/ROOT/nav.adoc

```diff
@@ -22,5 +22,7 @@
 *** xref:functions/dev_guide/python/develop-python.adoc[Developing Python functions]
 // Quarkus
 *** xref:functions/dev_guide/quarkus/develop-quarkus.adoc[Developing Quarkus functions]
+// Internal registry

```

Comment:
Is this the term we want to use in our product docs? I wonder if something like "In Cluster Registry" would be more descriptive.

Reply:
@lance Let me know if the wording can be further improved.